pkg/api/controllers/swagger.yaml

```diff
@@ -365,12 +387,11 @@ paths:
   /{ledger}/stats:
     get:
       tags:
-        - Stats
+        - Ledger
```

Comment:
Keep in ming this change break the sdk.
Ok for me on this one as this tag in confusing.
We have speak about them yesterday with @altitude @flemzord .
Can we just all tags to "ledger"?
This will lead on sdk side do "client.LedgerApi.getXXX"